charts/incubator/palworld/values.yaml

```diff
@@ -70,6 +74,21 @@ workload:
               echo "Setting Game Port..."
               sed -i 's/\(PublicPort=\)[^,]*/\1{{ .Values.service.main.ports.main.port }}/g' ${cfgFile}
               echo "Set to [$(grep -Po 'PublicPort=[^,]*' ${cfgFile})]"
+              echo "Setting Server Name..."
+              sed -i 's/\(ServerName=\)[^,]*/\1{{ .Values.palworld.game.name }}/g' ${cfgFile}
+              echo "Set to [$(grep -Po 'ServerName=[^,]*' ${cfgFile})]"
+              echo "Setting Server Description..."
+              sed -i 's/\(ServerDescription=\)[^,]*/\1{{ .Values.palworld.game.description }}/g' ${cfgFile}
+              echo "Set to [$(grep -Po 'ServerDescription=[^,]*' ${cfgFile})]"
+              echo "Setting Server  Player Max Num..."
+              sed -i 's/\(ServerPlayerMaxNum=\)[^,]*/\1{{ .Values.palworld.game.max_players }}/g' ${cfgFile}
+              echo "Set to [$(grep -Po 'ServerPlayerMaxNum=[^,]*' ${cfgFile})]"
+              echo "Setting Server Password..."
+              sed -i 's/\(ServerPassword=\)[^,]*/\1{{ .Values.palworld.game.password }}/g' ${cfgFile}
+              echo "Set to [$(grep -Po 'ServerPassword=[^,]*' ${cfgFile})]"
+              echo "Setting Server Admin Password..."
+              sed -i 's/\(AdminPassword=\)[^,]*/\1{{ .Values.palworld.admin.password }}/g' ${cfgFile}
```

Comment:
Pretty sure this will break if the password contains any regex characters like `?` `^` etc